magmartfront/src/app/admin/pokemon/[id]/page.tsx

```diff
@@ -58,32 +60,135 @@ function Product({params}: {params: {id: string}}) {
         getdados();
     }, []);
 
+    const handleDelete = async (e: any) => {
+        DeletePokemon(pok_id)
+    }
+
+    const [patchedTierId, setPatchedTierId] = useState('');
+    const [patchedStock, setPatchedStock] = useState('');
+    const [patchedPrice, setPatchedPrice] = useState('');
+    const handleEditPokemon = async () => {
+        try {
+            // Dados do Pokémon a serem enviados para o backend
+            const convertedData = {
+                tier_id: parseInt(patchedTierId), // Convertendo para inteiro
+                price: parseInt(patchedPrice), // Convertendo para ponto flutuante
+                stock: parseInt(patchedStock) // Convertendo para inteiro
+            };
+
+            // Envia a requisição para editar o Pokémon
+            const response = await fetch(`http://localhost:3001/pokemon/${params.id}/update`, {
```

Comment:
Padronizar a chamada usando variável de ambiente.
```suggestion
            const response = await fetch(`${process.env.API_URL}/pokemon/${params.id}/update`, {
```